net/remote/remote-node.sh

```diff
@@ -154,20 +160,21 @@ local|tar)
     if [[ $skipSetup != true ]]; then
       args=(
         --bootstrap-leader-stake-lamports "$stake"
+        --bootstrap-leader-lamports "$fundNodesInGenesisBlock"
```

Comment:
This should be wrapped in a `if [[ -n $fundNodesInGenesisBlock ]]; then` block right?

Reply:
yes